data/baibhav-tiwari.json

```diff
@@ -18,8 +24,52 @@
     },
     {
       "name": "Linkedin",
-      "url": "https://www.linkedin.com/in/baibhavtiwari/",
+      "url": "https://www.linkedin.com/in/baibhavtiwari",
       "icon": "FaLinkedin"
+    },
+    {
+      "name": "Hashnode",
+      "url": "https://hashnode.com/@baibhavv",
+      "icon": "SiHashnode"
+    }
+  ],
+  "milestones": [
```

Comment:
You added duplicate Milestones code. Remove one. You can check the full profile example here: https://linkfree.eddiehub.io/docs/full-profile-example